README.md

```diff
@@ -1,4 +1,4 @@
-# Strawberry GraphQL Django extension
```

Comment:
extenstion has a name clash with  strawberry schema extenstions IMO it is better this way.

Reply:
Is that O.K with you?